SPECS/etcd/etcd-3.4.3.spec

```diff
@@ -74,6 +74,8 @@ rm -rf %{buildroot}/*
 %config(noreplace) %{_sysconfdir}/etcd/etcd-default-conf.yml
 
 %changelog
+*   Mon Apr 26 2021 Nicolas Guibourge <nicolasg@microsoft.com> 3.4.3-2
+-   Increment release to force republishing using golang 1.15.11.
```

Comment:
Why are we incrementing these everywhere? Are these packages linking to some `golang` libs and the libs version changed?

Reply:
these packages have a build require dependency on golang...

Reply:
if I recall correctly etcd is written in go

Reply:
But it doesn't seem to have any run-time dependency on its libraries (if `golang` even provides any). It's not a problem to rebuild these other packages, it's just surprising. We didn't do that when we updated other packages as far. For instance, we do update `glib` or `gcc` every now and then, but don't update all specs relying on them. Do you think there's a reason we should be doing that and we're missing this from our process?

Reply:
Thanks for the clarification, Nicolas! I thought about it more and see your point.